modules/test/playwright/tests/export-import-web/instance.import.spec.ts

```diff
@@ -478,3 +481,310 @@ test('Can/not view Import menu item in Application menu depending on permissions
 		companyExportImportPage.exportImportPage.newImportButton
 	).toBeHidden();
 });
+
+test(
+	'can import custom object entries with original creator, and creator user does exist in the current environment',
+	{
+		tag: '@LPD-43217',
+	},
+	async ({
+		apiHelpers,
+		applicationsMenuPage,
+		companyExportImportPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const newObjectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFolderExternalReferenceCode: 'default',
+				status: {code: 0},
+			});
+
+		const user = await apiHelpers.headlessAdminUser.postUserAccount();
+
+		userData[user.alternateName] = {
+			name: user.givenName,
+			password: 'test',
+			surname: user.familyName,
+		};
+
+		let roles = await apiHelpers.headlessAdminUser.getRoles('Power User');
+
+		await apiHelpers.headlessAdminUser.postRoleUserAccountAssociation(
+			roles.items[0].id,
+			Number(user.id)
+		);
+
+		roles = await apiHelpers.headlessAdminUser.getRoles('Administrator');
+
+		await apiHelpers.headlessAdminUser.postRoleUserAccountAssociation(
+			roles.items[0].id,
+			Number(user.id)
+		);
+
+		await performLogout(page);
+
+		await performLogin(page, user.alternateName);
+
+		await applicationsMenuPage.goToObjects();
+		await viewObjectDefinitionsPage.clickEditObjectDefinitionLink(
+			newObjectDefinition.name
+		);
+		await page.getByLabel('Panel Link', {exact: true}).click();
+		await page.getByRole('option', {name: 'Object'}).click();
+		await page.getByRole('button', {name: 'Save'}).click();
+		await applicationsMenuPage.goToObjectDefinition(
+			newObjectDefinition.name
+		);
+		await page.getByText('Add ' + newObjectDefinition.name).click();
+		await page.getByLabel('textField').fill('testText');
+		await page.getByRole('button', {name: 'Save'}).click();
+		await waitForAlert(
+			page,
+			'Success:Your request completed successfully.'
+		);
+
+		await applicationsMenuPage.goToObjectDefinition(
+			newObjectDefinition.name
+		);
+
+		const objectEntryId = await page
+			.locator('table tr:first-child td:first-child')
+			.innerText();
+
+		const exportFilePath = await companyExportImportPage.export(
+			newObjectDefinition.name + ' 1 Items'
+		);
+
+		const applicationName =
+			'c/' + newObjectDefinition.name.toLowerCase() + 's';
+
+		await apiHelpers.delete(
+			`${apiHelpers.baseUrl}${applicationName}/${objectEntryId}`
+		);
+
+		await performLogout(page);
+
+		await performLogin(page, 'test');
+
+		await companyExportImportPage.import(exportFilePath);
+
+		await applicationsMenuPage.goToObjectDefinition(
+			newObjectDefinition.name
+		);
+		await expect(
+			page.getByRole('cell', {
+				name: user.givenName + ' ' + user.familyName,
+			})
+		).toBeVisible();
+
+		const objectDefinitionAPIClient =
```

Comment:
we're declaring it in every test, is it possible to set it outside the tests and reuse among all where needed?

Reply:
@DaniSzimko